docs/docsite/rst/reference_appendices/YAMLSyntax.rst

```diff
@@ -233,8 +233,10 @@ value::
        A good guide to YAML syntax
    `Mailing List <https://groups.google.com/group/ansible-project>`_
        Questions? Help? Ideas?  Stop by the list on Google Groups
-   `irc.freenode.net <http://irc.freenode.net>`_
-       #ansible IRC chat channel and #yaml for YAML specific questions
+   `irc.libera.chat <https://libera.chat/>`_
+       #ansible IRC chat channel
+   `irc.freenode.net <https://freenode.net/>`_
```

Comment:
```suggestion
       #ansible IRC chat channel
   `irc.libera.chat <https://libera.chat/>`_
```
or something

Reply:
Hmmmm..... they weren't ready to decide whether to move over to libera.chat when I spoke to them last but it looks like all the important people are there now.